src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -87,18 +87,7 @@ const SendFunds = ({ initialValues, onClose, onNext, recipientAddress, selectedT
         </IconButton>
       </Row>
       <Hairline />
-      <GnoForm
-        formMutators={formMutators}
-        initialValues={initialValues}
-        onSubmit={handleSubmit}
-        validation={(values) => {
-          const selectedTokenRecord = tokens.find((token) => token.address === values?.token)
-
-          return {
-            amount: maxValue(selectedTokenRecord?.balance)(values.amount),
-          }
```

Comment:
what happened here?

Reply:
this looked like code duplication https://github.com/gnosis/safe-react/blob/b28ad26af546cf4e5bb95fb88d7ebe59d6990d5a/src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx#L233

it's working fine without this double validation